src/clients/users_auto.rb

```diff
@@ -129,13 +129,8 @@ def main
         @ret = Users.Read == ""
         Progress.set(@progress_orig)
       elsif @func == "Write"
-        Yast.import "Progress"
-        Users.SetWriteOnly(true)
-        @progress_orig = Progress.set(false)
-        @ret = Users.Write == ""
-        Progress.set(@progress_orig)
-      # Return if configuration  was changed
-      # return boolean
+        # NOTE: this code was moved to users_finish client, as users set up is
+        # done now in the 1st stage.
```

Comment:
@kobliha Yes, that's make sense. I think we should remove the `Write` action then. But @schubi2 will say :)